### ISSUE TITLE
Caklculate standard deviation for TestNoisySumGaussianRealAggregation

### ISSUE THREAD

**Issue by elharo** [prestodb/presto#22325]
In test TestNoisySumGaussianRealAggregation figure out what range of expected values we can safely set to to pass at least 5 sigmas. Currently set at +/- 6. At +/- 5 we were failing maybe once in  hundred runs or so, maybe 3-4 sigmas.

**Comment by duykienvp** [comment]
Acknowledged. Sorry for a late reply. I will work with Jonathan for this test

**Comment by jonhehir** [comment]
TLDR: I'd just use a big number like 50 SDs.

The issue in question is related to https://github.com/prestodb/presto/pull/20703#discussion_r1310692769, although apparently one of these slipped through. Unless I'm misreading something, the test was indeed for $\pm 5 \sigma$. I know there was some discussion in #22279 about what the standard deviation refers to here, so here's a quick explanation:

The agg function in question here, `noisy_sum_gaussian`, essentially calculates a sum, then adds random noise. The random noise is Gaussian (i.e., normal) shaped, with zero mean and a standard deviation (i.e., width) of whatever the caller requests. So if the true sum is 100, the function basically returns a random number from a bell curve centered at 100 with a standard deviation of $\sigma = \mathtt{noiseScale}$.

The most useful unit tests in this file are the deterministic tests, namely the things with `noiseScale=0` or that use a seeded random number generator. This test basically hits the `SecureRandom` code path, but it doesn't need to be particularly tight. For this reason, in the comment linked above, I recommended using a small `noiseScale` and allowing a tolerance of  $\pm 50 \sigma$.
